rpi-eeprom-update

```diff
@@ -370,15 +394,27 @@ those devices to stop working until after the system is reboot.
 
 FIRMWARE_RELEASE_STATUS
 
-Specifies the release status of the firmware to apply. The default is 'critical'
-which is the most stable production version. Alternatively, 'stable' or
-'beta' may be selected for development releases.
-A 'critical' update is the latest stable production version and is normally
-only updated after it has been tested via the 'beta' and then a 'stable' release.
+Specifies the release status of the firmware to apply. The default is 'critical'.
+
 Before selecting a firmware release directory this script checks whether there
 is a board revision specific variant e.g. critical-c03111. If present then the
 board-revision specific version is used in preference.
 
+Release status:
+
+critical: The latest production release plus important security or hardware compatibility bug fixes.
+
+stable: Contains new features have already undergone some beta testing.
+These are candidats for new production releases.
```

Comment:
candidats -> candidates

Reply:
That would need to be resolved via a PR to the script

Reply:
:+1:  #138